Production/python/eostools.py

```diff
@@ -48,16 +45,13 @@ def runEOSCommand(path, cmd, *args):
     I think we should really try and raise an exception in case of problems.
     should be possible as the return code is provided in the tuple returned by runner."""
     
-    lfn = eosToLFN(path)
-    pfn = lfnToPFN(lfn)
-    tokens = cmsIO.splitPFN(pfn)
+    tokens = splitPFN(path)
```

Comment:
I think the error is later on at lines 194-196. It works when replacing those lines with:

    pfn = lfnToPFN(path)
    if not isEOSFile(pfn):
        runEOSCommand(pfn,'mkdir','-p')